js/misc/paygManager.js

```diff
@@ -60,6 +63,25 @@ const DBusErrorsMapping = {
     DISABLED          : 'com.endlessm.Payg1.Error.Disabled',
 };
 
+// Title and description text to be shown in the periodic reminders.
+const NOTIFICATION_TITLE_FORMAT_STRING = _("Pay-as-you-go time will expire in %s");
+const NOTIFICATION_DETAILED_TEXT = _("Talk to your sales representative to buy a new code.")
+
+// This list defines the different instants in time where we would
+// want to show notifications to the user reminding that the payg
+// subscription will be expiring soon.
+//
+// It contains a list of integers representing the number of seconds
+// earlier to the expiration time when we want to show a notification,
+// which needs to be sorted in DESCENDING order.
+const notificationAlertTimesSecs = [
+    60 * 60 * 2, // 2 hours
```

Comment:
I think that this first warning is much too late.  I imagine it may take a day or two to get in touch with the sales representative, so I really think we should give at least a one day notice.  Unless we get direction otherwise from @nedrichards, I would add one more notification for ``60 * 60 * 24, // 1 day` and then add string processing for number of days.

Reply:
I'll add for **1 and 2 days** (based on your comment on phabricator, which I agree with), and change the string processing function.